packages/node_modules/@webex/plugin-meetings/src/locus-info/infoUtils.js

```diff
@@ -3,9 +3,9 @@ import {SELF_ROLES} from '../constants';
 
 const InfoUtils = {};
 
-InfoUtils.parse = (info, roles) => {
+InfoUtils.parse = (info, roles, isJoined) => {
   const parsed = {
-    policy: InfoUtils.parsePolicy(info),
+    policy: InfoUtils.parsePolicy(info, isJoined),
```

Comment:
issue: this isn't quite right. We always want to parse the roles correctly. We should use isJoined when we define userDisplayHints. Currently it always includes parsed.policy, but it should only do that when the user isJoined.

Reply:
modified